LICENSES.txt

```diff
@@ -3059,6 +3068,7 @@ MIT
   reactive-streams-1.0.4.jar
   slf4j-api-1.7.36.jar
   slf4j-api-2.0.11.jar
+  slf4j-api-2.0.9.jar
```

Comment:
All other versions are bumped, here we include an older instead. What gives?

Reply:
This isn't a mistake, these files are auto-generated, it just means that arrow-vector is transitively dependent on 2.0.9